data/json/items/tool/woodworking.json

```
@@ -77,6 +77,46 @@
     "use_action": [ "CHAINSAW_ON" ],
     "flags": [ "MESSY", "TRADER_AVOID", "NONCONDUCTIVE", "POWERED", "FRAGILE_MELEE" ]
   },
+  {
+    "id": "polesaw_off",
+    "type": "TOOL",
+    "name": { "str": "polesaw (off)", "str_pl": "polesaws (off)" },
+    "description": "This is a lumbering tool intended for trimming hedges and tree branches.  If loaded with gas, using this item will cause it to turn on, making it into an unwieldy polearm.",
+    "weight": "9525 g",
+    "volume": "2500 ml",
+    "longest_side": "275 cm",
+    "price": 9000,
```

```suggestion
    "price": 40000,
```
Price is in decimals, and I couldn't find a gas one for $90, $400 seems a bit more reasonable, but that's just my opinion.